### DISCUSSION TITLE
performance problem

### DISCUSSION THREAD

**Discussion by ttt0** [EssentialsX/Essentials#4412] [Q&A] [answered]
https://timings.aikar.co/?id=02eb0db18f794dadbcc7198e8cb10537

![image](https://user-images.githubusercontent.com/66686400/127707153-a1bb724b-86ab-4b10-b1c5-12b788ca5ef5.png)

what can i do about it?

**Comment by Chew** [comment, marked as answer]
Nothing seems out of the issue based on the screenshot (everything takes an indescribably small amount of time to process). Your server is at 20 TPS with no signs of lag, there's nothing to worry about.
